armi/bookkeeping/db/database.py

```diff
@@ -1634,3 +1638,19 @@ def collectBlockNumberDensities(blocks) -> Dict[str, np.ndarray]:
         dataDict[nb.getDatabaseName()] = nucDensityMatrix[:, ni]
 
     return dataDict
+
+
+def _applyComponentNumberDensitiesMigration(comps, unpackedData):
```

Comment:
It's a little strange this isn't a staticmethod, seeing as it is only used in one class. And it can't be called from other files because it's private.